Stream_support/test/Stream_support/CMakeLists.txt

```diff
@@ -8,42 +8,23 @@ cmake_minimum_required(VERSION 3.1)
 
 find_package(CGAL QUIET)
 if ( CGAL_FOUND )
-  set(3MF_INCLUDE_DIR "" CACHE PATH "Path to lib3MF headers")
-  set(3MF_LIBRARY_DIR "" CACHE PATH "Path to lib3MF library files")
+  find_path(3MF_INCLUDE_DIR
+                NAMES Common Model
+                DOC "Path to lib3MF headers"
+               )
   if(IS_DIRECTORY "${3MF_INCLUDE_DIR}/Common")
     set(3MF_FOUND true)
   endif()
-  
+  find_library(3MF_LIBRARIES NAMES lib3MF DOC "Path to the lib3MF library")
```

Comment:
Use `3MF` instead of `lib3MF`, and CMake will be smart enough to search for `lib3MF.so` on Linux, and `3MF.lib` on Windows.